runelite-client/src/main/java/net/runelite/client/plugins/inventoryviewer/InventoryViewerOverlay.java

```diff
@@ -61,12 +63,21 @@ private InventoryViewerOverlay(Client client, ItemManager itemManager, Inventory
 		panelComponent.setOrientation(ComponentOrientation.HORIZONTAL);
 		this.itemManager = itemManager;
 		this.client = client;
+		this.config = config;
 		this.hidden = config.hiddenDefault();
 	}
 
 	@Override
 	public Dimension render(Graphics2D graphics)
 	{
+		if (config.hiddenInventory())
```

Comment:
Move this check after the `if (hidden)` check, as calling config is more expensive than checking a boolean field.